### PR TITLE
Lodash es

### PR DESCRIPTION
**Pull request by AmitJoki**
Migrates from CommonJS `lodash` to ES Module `lodash-es`.

### CODE REVIEW

**Comment by ragesoss** [comment]
It will be good to include a bundle size before/after for each of these, so we can see how much of an effect it's having. (I assume since this is just adding lodash-es but not completely removing lodash, there's no improvement yet?)

**Comment by AmitJoki** [comment]
@ragesoss yes. Also I haven't completely replaced all of them. Once I completely remove it, I will compare the stats with master and with this branch and put that info here.

**Comment by AmitJoki** [comment]
We have reduced roughly 50% of the lodash bundle size from **42 KB to 21 KB.**

File: package.json [hunk 1]
```diff
@@ -156,5 +137,6 @@
   },
   "resolutions": {
     "babel-core": "7.0.0-bridge.0"
-  }
+  },
+  "sideEffects": false
```
Comment (ragesoss) [inline package.json:141]: Please explain what this does (ideally, in an updated commit message).

File: webpack.config.js [hunk 2]
```diff
@@ -51,6 +52,12 @@ module.exports = (env) => {
     path: ['^.*css.*\\.js$']
   }));
 
+  plugins.push(new LodashModuleReplacementPlugin({
```
Comment (ragesoss) [inline webpack.config.js:55]: Please explain what these do (ideally, in an updated commit message).

**Pull request merged**